### ISSUE TITLE
CORS issue is there while clicking on submit/send button

### ISSUE THREAD

**Issue by manishverma10** [adrianhajdin/project_chat_application#84]
Access to XMLHttpRequest at 'https://project-chat-application.herokuapp.com/socket.io/?EIO=3&transport=polling&t=OCcR6zd' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

I you could fix this it

**Comment by leo-1228** [comment]
Yes, I can. please give me assign. If you want to discuss with me, this is my telegram @leo1228codeforce , contact me.
